code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4895,6 +4895,12 @@
 	nutriment_factor = FOOD_METABOLISM
 	color = "#664300" //rgb: 102, 67, 0
 
+/datum/reagent/drink/silencer/on_mob_life(var/mob/living/M)
+
+ 	if(..())
+            	 return 1
```

Comment:
yeah, that seems to be it
somehow i used spaces instead of tabs, how embarrasing